Lib/SharedSet.hpp

```diff
@@ -430,7 +430,7 @@ class SharedSet {
   }
   static unsigned hash(const T* arr, size_t len)
   {
-    static_assert(std::is_arithmetic<T>::value, "T must be safely hashable");
+    // static_assert(std::is_arithmetic<T>::value, "T must be safely hashable");
```

Comment:
Uh-oh. Did this fail?

Reply:
Ah yes. A pointer should also be safely hashable, right? Then I can add a disjunct with `std::is_pointer`.

Reply:
Yes please!